src/Evolution/Systems/GeneralizedHarmonic/Characteristics.cpp

```diff
@@ -0,0 +1,191 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+// IWYU pragma: no_include <array>
```

Comment:
What is the purpose of this line?

Reply:
IWYU complains that `Characteristics.cpp` needs to include `<array>` for the use of `array<>::v...`. I am not a 100% sure where is this being implicitly used (i suspect it could be the calls to `tensor.get`), but since the code in this file is not using it explicitly, I suppress this warning.